simulariumio/file_converter.py

```diff
@@ -82,5 +82,8 @@ def update_trajectory_info_version(self, data: Dict[str, Any]) -> Dict[str, Any]
                 "fovDegrees": DEFAULT_CAMERA_SETTINGS.FOV_DEGREES,
             }
             data["trajectoryInfo"]["version"] = 2
```

Comment:
shouldn't this also now be v = 3?

I think these (heere and line 87) should be `data["trajectoryInfo"]["version"] = self.current_trajectory_info_version`

Reply:
No, this block updates 1 to 2, then lines 85-87 update 2 to 3. Guess I could combine them but this gives me a place to comment that there are no mandatory fields added, and keeps the structure consistent with transitions that will need defaults added.

Reply:
oh ok, yeah, you should definitely have a block that checks for v == 2, like you say for clarity. I just didn't read this carefully enough and was assuming the other checks were `if else`s. You could still have the separate check for v == 2, but still update v1 ==> v3 data, but it doesn't really matter either way.